tru_tags.php

```diff
@@ -796,12 +796,6 @@ function tru_tags_admin_update_prefs() {
 
 
 function tru_tags_upsert_pref($name, $value) {
-	if (!safe_query('create table if not exists ' . safe_pfx('tru_tags_prefs').
-		' (name varchar(255) primary key, '.
-		'value varchar(255) not null)')) {
-		return 'Serious error - unable to create the ' . safe_pfx('tru_tags_prefs') . ' table.';
-	}
-
```

Comment:
Checking and creating a preference table is done at the [beginning](https://github.com/makss/tru-tags/blob/7108872118ba30d5acbeacc87d4fc961ed674114/tru_tags.php#L67) of the plugin, that is, before the function call `tru_tags_upsert_pref()`.